src/NUnitFramework/tests/Assertions/AssertThrowsAsyncTests.cs

```diff
@@ -0,0 +1,214 @@
+﻿using System;
+using System.Threading.Tasks;
+using NUnit.TestUtilities;
+
+namespace NUnit.Framework.Tests.Assertions
+{
+    [TestFixture]
+    public class AssertThrowsAsyncTests
+    {
```

Comment:
These would break some of our builds and so needs to be protected by an #if statement. 

Note that our general practice is to include framework files in all the framework builds but #if them out where not wanted. That facilitates searching the projects for all the #if statements.

This is the first point where I noticed it but it applies to other files as well.